Automattic/wp-calypso
# Feature: Jetpack AI Logo generator

This PR stands as a feature branch for the Jetpack AI Logo generator

Related to p1HpG7-qjH-p2

## Proposed Changes

* Adds the Jetpack AI Logo Generator feature to the `jetpack-ai-calypso` package.
* Adds the feature to the quick links section of My Home.

## Testing Instructions

For feature testing, see pedxs5-n0-p2.
The feature is contained in a new package and only imported in the quick links section of My Home, so check if there is no build errors or performance issues and nothing else obviously breaks.

## Screenshots

![2024-01-31_16-11-57](https://github.com/Automattic/wp-calypso/assets/8486249/45fb7868-9b34-4819-b393-5992d73930dc)
![2024-01-31_16-15-46](https://github.com/Automattic/wp-calypso/assets/8486249/fd188c4f-0f1b-4c2d-91a9-c37a3897c6d3)
![2024-01-31_16-21-47](https://github.com/Automattic/wp-calypso/assets/8486249/3fe4e080-ea85-4ab8-ba30-89940eab99d4)
![2024-01-31_16-22-17](https://github.com/Automattic/wp-calypso/assets/8486249/961ef1ae-bc47-41bf-9c6f-f6d0d428da8b)
![2024-01-31_16-26-38](https://github.com/Automattic/wp-calypso/assets/8486249/979a8055-6947-4e37-a226-1bfd336e3299)
![2024-01-31_16-30-51](https://github.com/Automattic/wp-calypso/assets/8486249/51376b3e-856c-4c29-8f69-a767ca779555)
![2024-01-31_16-31-19](https://github.com/Automattic/wp-calypso/assets/8486249/2b0f6b24-c4b0-41f0-ad8a-a68e3e2a7042)
![2024-01-31_16-34-18](https://github.com/Automattic/wp-calypso/assets/8486249/b8b2e405-32d2-4598-9e06-c20e569ea79d)
![2024-01-31_16-37-03](https://github.com/Automattic/wp-calypso/assets/8486249/e07177be-654e-4a42-b506-f55394a768b7)
![2024-01-31_16-38-19](https://github.com/Automattic/wp-calypso/assets/8486249/4c081f84-e3ca-46f6-9e75-451bde8ada12)
![2024-01-31_16-39-10](https://github.com/Automattic/wp-calypso/assets/8486249/29e17be6-0731-4180-8cca-fb922d45c45b)
![2024-01-31_16-40-04](https://github.com/Automattic/wp-calypso/assets/8486249/b681edd5-dcea-4ad5-a9f8-330066a4f48e)
![2024-01-31_16-41-02](https://github.com/Automattic/wp-calypso/assets/8486249/29a74637-503a-4fb3-b9ca-9ef37dc73746)
![2024-01-31_16-41-53](https://github.com/Automattic/wp-calypso/assets/8486249/3a688d34-6a96-488b-a776-0890e642983b)
![2024-01-31_16-42-54](https://github.com/Automattic/wp-calypso/assets/8486249/ea39dc12-555c-4471-b698-26e4619a2b35)

## Pre-merge Checklist
